css/panel-manager-style.css

```diff
@@ -0,0 +1,49 @@
+.panel-manager {
+  display: none;
+}
+
+#btnToggle {
+  position: fixed;
+  top: 70px;
+  right: 208px;
+  z-index: 100;
+}
+
+#btnToggle button {
+  border-radius: 4px 0 0 4px;
+  height: 50px;
+  width: 42px;
+}
+
+#slider {
+  background-color: white;
+  border: solid 1px #ccc;
+  border-radius: 0px;
+  border-bottom-left-radius: 4px;
+  border-top-left-radius: 4px;
+  border-left-color: #E6E6E6;
+  height: auto;
+  width: 250px;
+  overflow-y: scroll;
+  position: fixed;
+  top: 70px;
+  right: 0px;
+  bottom: 30px;
+  z-index: 10;
+}
+
+#tab-slider {
+  padding-left: 0px;
+}
+
+.sideways li:first-child {
+  display: none;
+}
+
+.sideways li:nth-child(2) {
```

Comment:
Because there exists a init item which is used to be cloned, the first panel we append would become the second child. There are some css style need to be revised for the first panel. That's why I call it by the second child.